src/arrayMethodReduce.js

```diff
@@ -5,7 +5,23 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator = initialValue;
+    let startIndex = 0;
+
+    if (initialValue === undefined) {
+      accumulator = 'undefined';
+    }
```

Comment:
you don't need it, if you didn't pass initialValue it's undefined already, you didn't define it so => undefined

Reply:
got it